### ISSUE TITLE
Listener not receiving events

### ISSUE THREAD

**Issue by dylanitorium** [stripe/stripe-cli#233]
### Issue
The command line listener is receiving no events, instead events are being sent to the dashboard.

(The CLI is also telling me to upgrade to 0.7.3 despite being on 0.7.3)

### Expected Behavior
The command line listener to receive events, and to not receive them in the dashboard.

### Steps to reproduce

1. `stripe login --interactive` (Then follow the steps, enter the secret key from the test API Keys)
2. `stripe listen`
3. Open a new terminal tab and run `stripe listen checkout.session.completed`

### Traceback
```
$ stripe listen
A newer version of the Stripe CLI is available, please update to: v0.7.3
> Ready! Your webhook signing secret is {...} (^C to quit)
```

```
$ stripe trigger checkout.session.completed
A newer version of the Stripe CLI is available, please update to: v0.7.3
Trigger succeeded! Check dashboard for event details.
```

### Environment
macOS
cli version 0.7.3

**Comment by dylanitorium** [comment]
Ok turns out that this was a websocket error:

```
 $ stripe listen --log-level=debug
A newer version of the Stripe CLI is available, please update to: v0.7.3
⣾ Getting ready... [Mon, 14 Oct 2019 13:37:36 NZDT] DEBUG stripeauth.client.Authorize: Authenticating with Stripe...
⣻ Getting ready... [Mon, 14 Oct 2019 13:37:46 NZDT] DEBUG stripeauth.Client.Authorize: Got successful response from Stripe display_connect_filter_warning=false reconnect_delay=61 websocket_authorized_feature=webhook-payloads websocket_id=cliws_{39296}_I1vO8R4pg84c1Fwy1jtLFyp0qsBI2AF1dWhWvjPkcC2xaLDfH5 websocket_url=wss://stripe-cli.stripe.com/subscribe/acct_1FRSL8JQCdQM28IU
[Mon, 14 Oct 2019 13:37:46 NZDT] DEBUG websocket.client.Run: Attempting to connect to Stripe
[Mon, 14 Oct 2019 13:37:46 NZDT] DEBUG websocket.Client.connect: Dialing websocket url=wss://stripe-cli.stripe.com/subscribe/acct_1FRSL8JQCdQM28IU?websocket_feature=webhook-payloads
> Ready! Your webhook signing secret is whsec_L0gSDZyIJnWTHTXCfFfu0wvDtau9A1G8 (^C to quit)
[Mon, 14 Oct 2019 13:37:56 NZDT] DEBUG websocket.Client.connect: Websocket connection error error=dial tcp: i/o timeout
[Mon, 14 Oct 2019 13:37:56 NZDT] DEBUG websocket.client.Run: Failed to connect to Stripe. Retrying...
```

The actual connection error looks to be due to some ISP configuration at my end, but it took me a decent while to even turn on the log level to debug given that it told me it was "Ready"

**Comment by ob-stripe** [comment]
Thanks for the report @dylanitorium. There are definitely a few things we can improve here, most notable the fact the CLI says "Ready!" before it's actually ready.
